spec/system/admin/reports/enterprise_fee_summaries_spec.rb

```diff
@@ -151,6 +155,8 @@
         find("#report_format").click
         select "CSV"
         click_on "Go"
+        perform_enqueued_jobs(only: ReportJob)
+        click_on "Download Report"
```

Comment:
We could create a helper for this but the same pattern isn't used anywhere else. Most specs display the report on screen.